js/core/index.js

```diff
@@ -24,6 +24,15 @@ var keyboard = require('./keyboard');
 var ps2 = require('./ps2');
 var pci = require('./pci');
 var net = require('./net');
+var stdio = require('./stdio');
+
+global.stdout = new stdio.stdout();
+global.stdin = new stdio.stdin();
```

Comment:
I think we shouldn't use globals, let's pass it as an argument to commands.
Curious why two objects instead of single stdio?